telegram/ext/commandhandler.py

```diff
@@ -119,6 +125,10 @@ def __init__(self,
             self.command = [command.lower()]
         else:
             self.command = [x.lower() for x in command]
+        for comm in self.command:
+            if not re.match(r'^[\da-z_]{1,32}$', comm):
```

Comment:
I'm not sure that 32 chars is a hard limit, dispite the docs... Do we care?

Both my phone and my desktop app, highlights commands that are longer than 32 chars at least.

Reply:
![image](https://user-images.githubusercontent.com/1409173/40325508-4c428504-5d3c-11e8-9360-f41f549e1338.png)
![image](https://user-images.githubusercontent.com/1409173/40325538-6bc21994-5d3c-11e8-9a91-dd7d6b3335f8.png)
There's a discrepancy here. @botfather won't let you add too long commands for suggestion, but they are recognised as entities. 

I vote to stay with the docs and enforce 32 chars